.cloudbuild/run_deploy.sh

```diff
@@ -0,0 +1,18 @@
+# /bin/bash
+
+gcloud run deploy ${_SERVICE} \
+    --platform managed --region ${_REGION} \
+    --image gcr.io/$PROJECT_ID/${_SERVICE}  \
+    --allow-unauthenticated  \
```

Comment:
Generally recommended to not include `--allow-unauthenticated` in your CD script.

Some considerations which may not apply to your use case or testing constraints:

* Requires broader privileges than regular Cloud Run deployment
* Removes the go/no-go decision from humans
* Locking down access is one step you might want to take in an emergency situation (e.g., data corruption), but this configuration will re-expose the service automatically.

If you do remove it, Cloud Run will persist this setting from revision to revision if it's used just a single time.

Reply:
I appreciate this comment and will consider the implications.